package com.example.currencyclientstarter;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

import java.util.Map;

@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class CurrencyApiResponse {

    @JsonProperty("data")
    private Map<String, CurrencyData> data;
}
